SillyBusinessInc/SillyBusinessGame
# Feat: #12 player healthbar

## Purpose of this PR:
Add a healthbar UI element to the game.

The FollowingEnemy has received a new AttackingState, this does configurable damage when the enemy is in a configurable range and has a configurable cooldown

## How to test:
Open up the PlayerHealthbarScene and hit run.
You should see 3 heart in the top left corner of the screen.
There is a testscript available on the TestScript GameObject, this allows you to do 1, 10 , or 100 damage to the player.

To test the FollowingEnemy attack simply run the game, and make sure to walk into the cone of vision of the enemy.
Once the enemy is in range it will do 25 damage (default, but configurable) and it should then not attack untill the cooldown is over. 

### links: 
closes #12 

### note:
This can be improved by making use of the soon to be merged event system, and will have to make use of the new stat system.
